lib/tessel/deployment/javascript.js

```diff
@@ -68,6 +68,17 @@ exportables.preBundle = function(opts) {
   return exportables.resolveBinaryModules(opts);
 };
 
+// exportables.preRun = function(tessel) {};
+
+exportables.postRun = function(tessel, remote) {
```

Comment:
It's not great that the arguments for `preRun`, `preBundle, `tarBundle`, and `postRun` all have very different arguments. Is there any way we can make them more consistent and intuitive?